app/views/docs/authentication-management.phtml

```diff
@@ -0,0 +1,397 @@
+
+val prefs = account.getPrefs()</code></pre>
+    </li>
+    <li>
+        <h3>Apple</h3>
+        <div class="ide" data-lang="swift" data-lang-label="Apple SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+
+let client = Client()
+    .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
+    .setProject("[PROJECT_ID]")                // Your project ID
+
+let account = Account(client)
+
+let prefs = try await account.getPrefs()</code></pre>
+    </li>
+    <li>
+        <h3>GraphQL</h3>
+        <div class="ide" data-lang="graphql" data-lang-label="GraphQL">
+            <pre class="line-numbers"><code class="prism language-graphql" data-prism>query {
+    accountGetPrefs {
+        data
+    }
+}</code></pre>
+        </div>
+    </li>
+</ul>
+
+<h2><a href="#users-api" id="users-api">Users API</a></h2>
+<p>
+    The Users API is a dedicated API for managing users from an admin's perspective. 
+    You'll notice that the Account API doesn't allow you to view or make changes to other users. 
+    This is by design and for <b>security reasons</b>.
+</p>
+
+<p>
+    You can use the Users API with an API key authenticated <a href="/docs/sdks">Server SDK</a> to manage users.
+    If you must expose parts of the Users API to normal users, we suggest doing so through an Appwrite Function.
+    Exposing API keys to users is <b>dangerous and a security risk</b>, by using an Appwrite Function, you can add your own validation to prevent malicious behavior.
+</p>
+
+<p>
+<a href="/docs//server/users"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> 
+    Learn more about the Users API
+</a>
+</p>
+
+<h2><a href="#labels" id="labels">Labels</a></h2>
+<p>
+    Labels are a good way to flag a user to grant them access to resources. 
+    For example, a <code>subscriber</code> label can be added to a user once they've purchased a subscription.
+</p>
+
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Node.js</h3>
+        <div class="ide margin-top-small" data-lang="nodejs" data-lang-label="Node.js SDK">
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>const sdk = require('node-appwrite');
+
+const client = new sdk.Client();
+
+const users = new sdk.Users(client);
+
+let res = await users.updateLabels('[USER_ID]', [ Role.label('subscriber') ]);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>PHP</h3>
+
+        <div class="ide margin-top-small" data-lang="php" data-lang-label="PHP SDK">
+            <pre class="line-numbers"><code class="prism language-php" data-prism>use Appwrite\Client;
+use Appwrite\Services\Users;
+use Appwrite\Role;
+
+$client = new Client();
+
+$client
+    .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2') // Your project ID
+    .setKey('919c2d18fb5d4...a2ae413da83346ad2') // Your secret API key
```

Comment:
I think we can use typical placeholders here

Reply:
Let's check all places

Reply:
We use '919c2d18fb5d4...a2ae413da83346ad2' all over the place, I think it's fine. We historically found that people didn't know that a key looks like a giant hash